client/client.go

```diff
@@ -369,6 +369,18 @@ type RemoteContext struct {
 	// MaxConcurrentDownloads is the max concurrent content downloads for each pull.
 	MaxConcurrentDownloads int
 
+	// MaxConcurrentDownloadOperations is the max number of operation that can
```

Comment:
These are defined but not used. Do you plan to add support for client opts as a follow up?

Reply:
not at all, an artefact from when I wasn't sure if these params needed to go there. Removing. I can add support for the client if you think it's worth it though

Reply:
I think this will be useful to have specially for clients that do not use transfer service for image related operations. Though this can be done as a follow up PR and need not be part of this PR.

Reply:
Gotcha, awesome, yes, I can do a follow up 😊

Reply:
Okay, will spend some time to do this one.

Reply:
Hello, I re-added client support, please lmk what you think !

With this addition, I've been able to pull with these settings through `crictl`